example/cluster.js

```diff
@@ -0,0 +1,28 @@
+var sticky = require('../');
+var cluster = require('cluster');
+var port = 3000;
+var http = require('http');
+// var io = require('socket.io');
```

Comment:
no comments for currently not used code please. if you need it again, it is already in an older commit waiting for you.